.codecov.yml

```diff
@@ -27,3 +27,4 @@ comment:
 
 ignore:
   - "*Errors.go"  # ignore error files
+  - "file.go"
```

Comment:
I just did this because I don't have time to write json tests for every batch type, and I'm not sure how worth doing that is.